### PR TITLE
Dynamic static model pipeline bug fixes and new part table implementation for center surround dynamic static chain

### PR DESCRIPTION
**Pull request by ZhuokunDing**
1. Implement `DvScanConfig.Nn10` to import static image responses from `dynamic_vision.nns` version 10
2. Deprecate `InputConfig.NeuroStaticFrame` due to bugs in computing statistics. Correct the computation of statistics in `StatsConfig.NoBehFullFieldFrame` by skipping the abundant input reshaping step.
3. Implement `NeuroStaticValidFrame`. Only import images that were shown during valid trials in the static scan.
4. Implement `DvScanConfig.Nn10Frame2` to import dynamic model responses to apertured static images from `dynamic_vision.nns` version 10
5. Implement `StatsConfig.NoBehFullFieldFrame2` to compute stats for `stimulus.Frame2` scans

### CODE REVIEW

File: neuro_data/static_images/dataset_config.py [hunk 1]
```diff
@@ -63,6 +71,91 @@ def input(self, scan_key):
...
+                    and frame.shape[1] == params["row"]
+                    and frame.shape[2] == params["col"]
+                ), "dimension mismatch, only support 3-D frame (B,H,W)"
+                # adjust frame shape to (B,1,W,H)
+                frame = frame[:, None, ...]
+            else:
+                raise NotImplementedError(
+                    f'InputConfig: gamma={params["gamma"]}, linear_mon={params["linear_mon"]} not implemented!'
+                )
+            return trial_idx, cond, frame, np.full(len(trial_idx), "stimulus.Frame")
```
Comment (jiakunf) [inline neuro_data/static_images/dataset_config.py:117]: remove hard-coded stimulus_type.

Comment (ZhuokunDing) [inline neuro_data/static_images/dataset_config.py:117]: Avoid hardcoding stimulus types, fetch them from stimulus.Condition table instead.

File: neuro_data/static_images/dataset_config.py [hunk 2]
```diff
@@ -309,6 +398,105 @@ def stats(self, condition_hashes, images, responses, tiers):
...
+
+            data_mean = wstats(data_flat, data_mask).mean
+            data_std = wstats(data_flat, data_mask).std
+
+            ret["stimulus.Frame2"] = dict(
+                mean=data_mean.astype(np.float32),
+                std=data_std.astype(np.float32),
+                min=data.min().astype(np.float32),
+                max=data.max().astype(np.float32),
+                median=np.median(data).astype(np.float32),
```
Comment (jiakunf) [inline neuro_data/static_images/dataset_config.py:459]: median isn't computed with correct masking. downstream computation shouldn't use this value.

File: neuro_data/static_images/dataset_config.py (hunk 1 above)
Comment (ZhuokunDing) [reply neuro_data/static_images/dataset_config.py]: `stimulus_type` is now fetched from `stimulus.Condition`

File: neuro_data/static_images/dataset_config.py [hunk 3]
```diff
@@ -63,6 +71,91 @@ def input(self, scan_key):
...
+                ), "dimension mismatch, only support 3-D frame (B,H,W)"
+                # adjust frame shape to (B,1,W,H)
+                frame = frame[:, None, ...]
+            else:
+                raise NotImplementedError(
+                    f'InputConfig: gamma={params["gamma"]}, linear_mon={params["linear_mon"]} not implemented!'
+                )
+            return trial_idx, cond, frame, np.full(len(trial_idx), "stimulus.Frame")
+
+    class NeuroStaticFrameV2(dj.Part):
```
[the remark this answers is not in the document]
Comment (ZhuokunDing) [reply neuro_data/static_images/dataset_config.py]: Removed.

File: neuro_data/static_images/dataset_config.py (hunk 2 above)
Comment (ZhuokunDing) [reply neuro_data/static_images/dataset_config.py]: Comment added.

**Pull request merged**